schemas/TransferTemplate.json

```diff
@@ -65,6 +61,21 @@
       "minItems": 1,
       "uniqueItems": true,
       "items": { "$ref": "FundsTemplate.json" }
+    },
+    "additional_info": {
+      "description": "Includes part_of_payment, case, etc.",
+      "type": "object",
+      "properties": {
+        "part_of_payment": {
```

Comment:
I think clear abstraction boundaries are a higher priority. Here are a couple of reasons why:
1. In the future we may want to do transfers that are not part of a payment. Then we would probably end up adding a second field called something like "part_of_some_other_operation" and meanwhile "part_of_payment" remains in the schema unused.
2. A developer or user may see "part_of_payment" in a response object and assume "oh, so i can add logic to the internal ledger for keeping track of payments" even though this is counter to the design goal. Or they might just be confused by why the ledger cares about payments.
These are just examples, the real justification is maintaining clear boundaries.